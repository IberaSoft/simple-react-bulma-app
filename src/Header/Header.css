.header {

    background-color: #222;
    color: white;
    grid-area: header;
    height: 170px;
    padding: 20px;
    text-align: center;
}

.header-title {
    font-size: 1em;
}

.header-logo {
    display: inline-block;
    vertical-align: middle;
    width: 7%;
}

.logo {
    animation: logo-spin infinite 20s linear;
    height: 80px;
    width: 100%;
}

@keyframes logo-spin {
    from { transform: rotate(0deg); }
    to { transform: rotate(360deg); }
}