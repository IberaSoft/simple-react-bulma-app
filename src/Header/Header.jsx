import React from 'react';
import logo from './../Assets/logo.svg';
import Navigation from './../Navigation/Navigation';
//css
import './Header.css';

const Header = (props) => (
    <header className="navbar is-white topNav">
        <div className="container">
            <div className="navbar-brand">
                <img src={logo} className="logo" width="150" height="150" alt="logo" />
            </div>
            <Navigation className="navbar-end" activeIndex={props.activeIndex} changeIndex={props.changeIndex} />
        </div>
    </header>
)

export default Header