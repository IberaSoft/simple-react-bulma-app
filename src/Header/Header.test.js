import React from 'react'
import Header from './Header'
import ShallowRenderer from 'react-test-renderer/shallow';

describe('Header component', () => {
    it('Should match the snapshot', () => {
        const renderer = new ShallowRenderer();
        const result = renderer.render(<Header />);
        expect(result).toMatchSnapshot();
    });
})