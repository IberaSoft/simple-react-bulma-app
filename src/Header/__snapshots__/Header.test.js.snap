// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should match the snapshot 1`] = `
<header
  className="navbar is-white topNav"
>
  <div
    className="container"
  >
    <div
      className="navbar-brand"
    >
      <img
        alt="logo"
        className="logo"
        height="150"
        src="logo.svg"
        width="150"
      />
    </div>
    <Navigation
      activeIndex={undefined}
      changeIndex={undefined}
      className="navbar-end"
      menuItems={undefined}
    />
  </div>
</header>
`;
